docs/cli-tool.md

```diff
@@ -0,0 +1,38 @@
+# CLI Tool
+
+[jvsctl](../cmd/jvsctl) facilitates the justification verification flow provided
+by abcxyz/jvs
+
+## Usage
+
+jvsctl [command]
+
+### Global Flags:
+
+*   `--config` : string <br />
+    config file (default is $HOME/.jvsctl/config.yaml) <br />
+*   `-h`, `--help` <br />
+    help for jvsctl <br />
+*   `--insecure` : bool <br />
+    use insecure connection to JVS server <br />
+*   `--server` : string <br />
+    overwrite the JVS server address
+
+### jvsctl token [flags]
+
+To generate a justification token
+
+#### Flags
+
+`--breakglass` : bool <br />
+Whether it will be a breakglass action <br />
+`-e`, `--explanation` : string <br />
+The explanation for the action <br />
+`--ttl` : duration <br />
+The token time-to-live duration (default 1h0m0s) <br />
+
+#### Example
+
+```shell
+jvsctl token --explanation "issues/12345" -ttl 30m
+```
```

Comment:
More examples will be helpful, e.g. breakglass

Reply:
And add description what the command does per example.
Also add a section for how to use with lumberjack.

Reply:
I thought how to use lumberjack will be written in the lumberjack repo. What info do we want to include here in jvsctl regarding how to use with lumberjack?

Reply:
description and breakglass example added.

Reply:
Discussed offline. Will add a section for how to use with lumberjack later after we finish the refactor code change and doc update in lumberjack: https://github.com/abcxyz/lumberjack/pull/279, https://github.com/abcxyz/lumberjack/issues/280 and https://github.com/abcxyz/lumberjack/issues/283